valhalla/mjolnir/util.h

```diff
@@ -42,6 +42,7 @@ enum class BuildStage : int8_t {
 constexpr uint8_t kMinor = 1;
 constexpr uint8_t kStopSign = 2;
 constexpr uint8_t kYieldSign = 4;
+constexpr uint8_t kMaxCurvature = 15;
```

Comment:
This should probably go in `dynamiccost.h`

Reply:
Ah now I see what you meant, I wasn't paying full attention... I see it's in `graphconstants.h` now, so from there we should be able to use it in both mjolnir and sif. Maybe you were missing an include in mjolnir?

But yeah, I was just thinking about whether we shouldn't open up the other use case as well while at it, but open to hear other people's opinion on this!